### ISSUE TITLE
Any way to utilize coconut within a .RMD chunk?

### ISSUE THREAD

**Issue by dylanjm** [evhub/coconut#330]
Brand new to Coconut and already loving it. I'm currently taking a numerical analysis class where we are required to do all our homework in Python while using LaTeX. I find that Rstudio is the best way I can manage LaTeX and code at the same time.

I would love to use Coconut for this class and have seen that there is already jupyter notebook integration. Is there anyway that I could use coconut in my .Rmd files?

**Comment by evhub** [comment]
@dylanjm I don't think so, though that's potentially something we could look into adding. What's the format of `.rmd` files?

**Comment by dylanjm** [comment]
The format of `.rmd` files are markdown files with the ability run R code and now most recently Python code. I wouldn't have asked if the Python ability wasn't available but I was wondering if we could leverage the Python compilation to use Coconut. 

I'm not an extreme programmer by any means, so I'm unsure if this is even possible. But it would be super useful in my reports. 

I know I could use Jupyter but I use R a lot and really like being able to convert to LaTeX for my reports.

**Comment by evhub** [comment]
@dylanjm Theoretically, you could do something like
```
from coconut.convenience import parse
exec(parse("""
<coconut code here>
"""))
```
though that is definitely less than ideal. If all you want is LaTeX exporting, though, Jupyter notebooks [can do that](https://ipython.org/ipython-doc/3/notebook/nbconvert.html).
